bucket/qemu.json

```diff
@@ -16,6 +16,74 @@
     "pre_install": "Expand-7zipArchive \"$dir\\dl.7z_\" \"$dir\" -Removal -Switches '-xr!*.exe.nsis -x!$PLUGINSDIR'",
     "env_add_path": ".",
```

Comment:
If we're adding all the .exes, shouldn't we remove the `"env_add_path": ".",` now?

Reply:
You should try restarting the terminal for running the commands.

Reply:
Oh yes, and rebooting does makes it work. thx :)